src/aggregator/aggregator/follower_flush_mgr.go

```diff
@@ -187,6 +188,13 @@ func (mgr *followerFlushManager) Prepare(buckets []*flushBucket) (flushTask, tim
 		mgr.flushMode = kvUpdateFollowerFlush
 		flushersByInterval = mgr.flushersFromKVUpdateWithLock(buckets)
 		needsFlush = true
+		for _, flushers := range flushersByInterval {
+			for _, flusher := range flushers.flushers {
+				lag := nowNanos - flusher.flushBeforeNanos
+				flushers.followerFlushLag.RecordDuration(time.Duration(lag))
```

Comment:
this seems kind of early to record the lag, since it's just being prepared. i'd be more interested in recording it when the flusher actually runs.

Reply:
The other histogram should record the lag when it runs, this one should just show how far behind the bucket is when it's eligible for a flush